### ISSUE TITLE
I cannot seem to change the module shape in HX.

### ISSUE THREAD

**Issue by Chronotides** [blowfishpro/B9-Aerospace#54]
Hello Blowfish and co,
I recently added b9 and b9 hx to my KSP 1.1.0 (using correct version, i just prefer without the comm system as I am still relatively new to the game, but will reinstall updated version when I am ready to do so.). However, I am trying to change the configuration of the HX4 series of modules, and cannot seem to bring up the menu on the structure parts by right-clicking. Is there another way to bring that menu up, or was the feature removed? I wish to change it to a fuel tank, but I am unable to do so, and therefore have to use infinite propellant when in sandbox mode, kind of defeating the purpose of the game.

To summarize, I wish to know how to turn the HX parts (under the structure tab) into their other configurations to use as fuel storage of various types.

Thank you,

Chronotides

**Comment by blowfishpro** [comment]
Make sure you have a compatible version of B9PartSwitch installed.  If the switcher isn't showing up that's the most likely problem.

If you still see the problem please read this post: http://forum.kerbalspaceprogram.com/index.php?showtopic=83212 - it will tell you how to find and upload KSP's log, which I will need to inspect to really see what the problem is

Also I'm pretty sure you can disable the effects of the comm system in the difficulty settings.
